.github/workflows/msal-node-e2e.yml

```diff
@@ -12,13 +12,20 @@ on:
     paths: 
       - 'lib/msal-node/**/*' 
       - 'lib/msal-common/**/*'
-      - 'samples/msal-node-samples/EndToEndTestApp/**/*'
+      - 'samples/msal-node-samples/**/*'
       - '!**.md'
+      - '.github/workflows/msal-node-e2e.yml'
 
 jobs:
   run-e2e:
     if: (github.repository == 'AzureAD/microsoft-authentication-library-for-js') && ((github.event.pull_request.head.repo.full_name == github.repository) || (github.event_name == 'push'))
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [ 12, 14 ]
```

Comment:
Add 10 to this too since we support it for now

Reply:
Done!